boot/bootchain.go

```diff
@@ -105,30 +105,14 @@ func stringListsLess(sl1, sl2 []string) bool {
 	return false
 }
 
-func toPredictableBootAsset(b *bootAsset) *bootAsset {
-	if b == nil {
-		return nil
-	}
-	newB := *b
-	if b.Hashes != nil {
-		newB.Hashes = make([]string, len(b.Hashes))
-		copy(newB.Hashes, b.Hashes)
-		sort.Strings(newB.Hashes)
-	}
-	return &newB
-}
-
 func toPredictableBootChain(b *bootChain) *bootChain {
 	if b == nil {
 		return nil
 	}
 	newB := *b
```

Comment:
What is the reason we need to assign `newB.AssetChain = b.AssetChain` when we copy b here? Wouldn't that make the next assignment superfluous?

Reply:
The reason is I am dumb ;)